test/MOI_wrapper.jl

```diff
@@ -12,6 +12,19 @@ import AmplNLWriter
 const MOI = AmplNLWriter.MOI
 
 function runtests(path)
+    path() do exe
+        lbt_default_libs = get(ENV, "LBT_DEFAULT_LIBS", AmplNLWriter._get_blas_libs())
+        if isempty(lbt_default_libs)
+            run(`$exe -v`)
+        else
+            run(addenv(`$exe -v`, "LBT_DEFAULT_LIBS" => lbt_default_libs))
+        end
+    end
+    path() do exe
+        run(`$exe -v`)
+    end
```

Comment:
@amontoison I think you need some tests like this.

In the normal tests, we can't open the executable, but we trap the error and return `OTHER_ERROR` to MOI so it looks as if the binary didn't find a solution, but really it crashed.